op-conductor/rpc/ws/flashblocks_handler.go

```diff
@@ -0,0 +1,278 @@
+	// Start the WebSocket server
+	if err := h.startWebSocketServer(ctx); err != nil {
+		return err
+	}
+
+	// Start the rollup boost listener
+	if err := h.startRollupBoostListener(ctx); err != nil {
+		return err
+	}
+
+	return nil
+}
+
+// Stop closes any open WebSocket connections and shuts down the server
+// This should only be called if the conductor is shut down
+func (h *Handler) Stop() {
+	// Close the rollup boost connection if it exists
+	if h.rollupBoostConn != nil {
+		h.rollupBoostConn.Close()
+		h.rollupBoostConn = nil
+	}
+
+	// Close all WebSocket client connections
+	h.wsClientMu.Lock()
+	for _, client := range h.wsClients {
+		client.Close()
+	}
+	h.wsClients = nil
+	h.wsClientMu.Unlock()
+
+	// Force close the HTTP server if it's running
+	if h.server != nil {
+		h.log.Info("Forcibly closing WebSocket server")
+		err := h.server.Close()
+		if err != nil {
+			h.log.Error("Error closing WebSocket server", "err", err)
+		}
+		h.log.Info("WebSocket server closed")
+	}
+}
+
+// BroadcastMessage sends a message to all connected WebSocket clients
+func (h *Handler) BroadcastMessage(message []byte) {
+	h.wsClientMu.Lock()
+	defer h.wsClientMu.Unlock()
+
+	if len(h.wsClients) == 0 {
+		h.log.Debug("no WebSocket clients connected, not broadcasting message")
+		return
+	}
+
+	h.log.Info("Broadcasting message to WebSocket clients", "clientCount", len(h.wsClients))
+
+	// Send message to all clients
+	for _, client := range h.wsClients {
+		err := client.WriteMessage(websocket.TextMessage, message)
+		if err != nil {
+			h.log.Error("error writing to WebSocket client", "err", err, "remote", client.RemoteAddr())
+			// Just log the error but keep the connection
+		}
+	}
+}
+
+// startRollupBoostListener initializes and starts a WebSocket client to listen for rollup boost messages
+func (h *Handler) startRollupBoostListener(ctx context.Context) error {
+	if h.cfg.RollupBoostWsURL == "" {
+		h.log.Info("rollup boost WebSocket disabled, no URL configured")
+		return nil
+	}
+
+	// Start a goroutine to maintain the rollup boost WebSocket connection
+	go func() {
+		backoff := time.Second
+		maxBackoff := 30 * time.Second
+
+		for {
+			select {
+			case <-ctx.Done():
+				if h.rollupBoostConn != nil {
+					h.log.Info("closing rollup boost WebSocket connection due to context cancellation")
+					h.rollupBoostConn.Close()
+					h.rollupBoostConn = nil
+				}
+				return
+			default:
+				// Try to connect if not connected
+				if h.rollupBoostConn == nil {
+					conn, _, err := websocket.DefaultDialer.Dial(h.cfg.RollupBoostWsURL, nil)
+					if err != nil {
+						h.log.Error("failed to connect to rollup boost WebSocket, will retry", "err", err, "backoff", backoff)
+						time.Sleep(backoff)
+						// After each failed connection attempt, increases the wait time by multiplying by 1.5
+						// Caps the maximum wait time at 30 seconds
+						backoff = time.Duration(min(float64(backoff)*1.5, float64(maxBackoff)))
+						continue
+					}
```

Comment:
Just try dial the connection 3 times, if failed, return error (remember this is during initialization, we can afford a failure, we'll know something's wrong and fix it).